.github/workflows/_build_artifacts.yml

```diff
@@ -254,6 +258,12 @@ jobs:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           ref: ${{ inputs.sha }}
+      - name: Authenticate to Google Cloud
```

Comment:
@thomaseizinger I purposefully inlined this step rather than adding it to `/.actions`. The latter causes dependabot to fail to keep this dep up to date, which I weigh as a larger downside than having more verbosity overhead in the workflow file.